packages/dispatchr/lib/DispatcherContext.js

```diff
@@ -37,7 +37,11 @@ DispatcherContext.prototype.getStore = function getStore(name) {
     if (!this.storeInstances[storeName]) {
         var Store = this.dispatcher.stores[storeName];
         if (!Store) {
-            throw new Error('Store ' + storeName + ' was not registered.');
+            if ('function' === typeof name) {
```

Comment:
do we have any env variable anywhere? if yes we might want to print an warning on dev mode